### PR TITLE
Fourth Order Equation with Exciter, AVR and Governor

### PR DESCRIPTION
**Pull request by WouterVer**
work in progress

### CODE REVIEW

**Comment by timkittel** [comment]
on the side: If you want your the automatic checks (ci = continuous integration) to pass, then add a short note to the [CHANGELOG.md](https://github.com/JuliaEnergy/PowerDynBase.jl/blob/master/CHANGELOG.md)with your changes and a link to this PR.

**Comment by SabineAuer** [comment]
@WouterVer : this looks very promising! As a side note: 4th-order equation means that I have 4 differential equations describing my node dynamics. In your case it is more than 10 equations (if I counted right) so I would suggest to not call this 4th-order equation but something like 10th-order equation with `TenthOrderEqGovernorExciterAVR`

**Comment by timkittel** [comment]
As far as I understand, the 4th order usually refers to the number of equations for the power generation of the synchronous machine. I.e. how strongly you reduce the field equations, e.g. [here for some examples](https://wiki.openelectrical.org/index.php?title=Synchronous_Machine_Models#6th_Order_.28Anderson-Fouad.29_Model). In this case, it is the 4th order model.

Additionally, the exiciter, avr and governor have been added to the 4th order model. Hence, I would stay with the electrical engineering naming of calling it 4th order plus the additional parts instead of using a novel nomenclature that might confuse people.

File: src/NodeDynamics/FourthOrderEquationGovernorExciterAVR.jl [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	\dfrac{dv_r}{dt} = \dfrac{1}{T_a} (-v_r + (K_a R_f) -\dfrac{K_a K_f}{T_f}e_{fd} + K_a (V_{ref} - abs(u_{terminal})))
+	\dfrac{de_{fd}}{dt} = \dfrac{1}{T_e} (-K_e + S_{e}(e_{fd})e_{fd} + v_r)
+```
+
+# Governor equations
+```math
+    \dfrac{dP_m}{dt} = \dfrac{1}{T_{ch}} (-P_m + P_{sv})
+    Assumption: T_m = P_m
+    \dfrac{dP_{sv}}{dt} = \dfrac{1}{T_{sv}} (-P_{sv} + P_c -\dfrac{1}{R_d} (\dfrac{\omega}{\omega_s} - 1))
+```
```
Comment (timkittel) [inline src/NodeDynamics/FourthOrderEquationGovernorExciterAVR.jl]: As just discussed, please fix the docs here.

**Pull request merged**